Description: emphasize that a hyphen is really needed in "-1" by writing it as \(hy1
Author: أحمد المحمودي (Ahmed El-Mahmoudy) <aelmahmoudy@users.sourceforge.net>
Bug: https://bugs.freedesktop.org/show_bug.cgi?id=54147
Forwarded: https://bugs.freedesktop.org/attachment.cgi?id=66207
Index: fribidi/doc/fribidi_remove_bidi_marks.3
===================================================================
--- fribidi.orig/doc/fribidi_remove_bidi_marks.3	2018-07-15 15:05:37.948747901 +0200
+++ fribidi/doc/fribidi_remove_bidi_marks.3	2018-07-15 15:07:22.761168734 +0200
@@ -54,14 +54,14 @@
 string, the other way. Moreover, the position maps should be filled with
 valid entries.
 
-A position map pointing to a removed character is filled with \-1. By the
+A position map pointing to a removed character is filled with \(hy1. By the
 way, you should not use embedding_levels if str is visual string.
 
 For best results this function should be run on a whole paragraph, not
 lines; but feel free to do otherwise if you know what you are doing.
 Deprecated.  Use fribidi_remove_special_chars instead.
 .SH "RETURNS"
-New length of the string, or \-1 if an error occurred (memory
+New length of the string, or \(hy1 if an error occurred (memory
 allocation failure most probably).
 .SH "SEE ALSO"
 fribidi_shape_arabic(3),
